activities/value-and-impact/policy-documents.md

```diff
@@ -13,8 +13,8 @@ explains: where we are mentioned in public policy documents
 
 * Netherlands national government: [Letter to parliament about the release of government software source code](https://www.rijksoverheid.nl/documenten/kamerstukken/2020/04/17/kamerbrief-inzake-vrijgeven-broncode-overheidssoftware) (in Dutch)
 * Province of South Holland: [Council decision on membership of the Foundation for Public Code](https://www.zuid-holland.nl/overons/bestuur-zh/gedeputeerde-staten/besluiten/2020/september/1-september/lidmaatschap-foundation-for-public-code-vereniging/) (in Dutch)
-* Amsterdam municipality: [Mid-term report of the A Digital City Agenda (Agenda Digitale Stad) [PDF]](https://amsterdam.raadsinformatie.nl/document/9725293/1/09012f9783374971) (in Dutch)
```

Comment:
This change looks like a mistake. It removes the punctuation that signals that the title is translated.